### PR TITLE
added option reportFile

### PR DESCRIPTION
**Pull request by kluvi**

### CODE REVIEW

**Comment by XhmikosR** [comment]
@kluvi: please squash the patches.

File: tasks/uncss.js [hunk 1]
```diff
@@ -52,7 +52,9 @@ module.exports = function ( grunt ) {
 
                     grunt.file.write( file.dest, output );
                     grunt.log.writeln('File ' + chalk.cyan( file.dest ) + ' created: ' + maxmin( report.original, output, options.report === 'gzip' ) );
-
+                    if(typeof(options.reportFile) !== 'undefined' && options.reportFile.length > 0) {
```
Comment (XhmikosR) [inline tasks/uncss.js]: A space after `if` is missing.

**Comment by kluvi** [comment]
Sorry.. first time contributing to other project. Commits squashed, space added.

**Comment by addyosmani** [comment]
Change LGTM

**Pull request merged**